uicomponent-compose/main/src/main/java/io/github/droidkaigi/feeder/viewmodel/ProvideRealViewModels.kt

```diff
@@ -12,5 +13,8 @@ fun ProvideViewModels(content: @Composable () -> Unit) {
         ProvideStaffViewModel(viewModel = fakeStaffViewModel()) {
             content()
         }
+        ProvideContributorViewModel(viewModel<RealContributorViewModel>()) {
+            content()
+        }
```

Comment:
Thank you for providing the viewmodel!
Maybe I should change it like this?

```suggestion
        ProvideStaffViewModel(viewModel = fakeStaffViewModel()) {
            ProvideContributorViewModel(viewModel<RealContributorViewModel>()) {
                content()
            }
        }
```

Reply:
I found my code was not working well.
seems your code is right!

Does ProvideHogeViewModel method continue to nest like above?
I mistakenly thought it would be like a screen hierarchy.

Reply:
It may be necessary to think about the structure a little, but I would like to implement it this way once. 🙏

Reply:
![image](https://user-images.githubusercontent.com/1386930/110212501-eed97400-7ede-11eb-9f2a-a7c76fe9124f.png)
For example, it looks like this. No need to fix now.

Reply:
I can imagine. Thank you 👍